src/openvic-simulation/map/ProvinceInstance.hpp

```diff
@@ -159,9 +160,11 @@ namespace OpenVic {
 		bool setup(BuildingTypeManager const& building_type_manager);
 		bool apply_history_to_province(ProvinceHistoryEntry const& entry, CountryInstanceManager& country_manager);
 
-		void initialise_for_new_game(ModifierEffectCache const& modifier_effect_cache);
+		void initialise_rgo(ModifierEffectCache const& modifier_effect_cache);
 
 		void setup_pop_test_values(IssueManager const& issue_manager);
+		State* get_mutable_state();
```

Comment:
This isn't needed anywhere right now, although it may be useful in the future so there's an argument for keeping it.